tasks/config/run.js

```diff
@@ -224,6 +194,55 @@ module.exports = function (grunt) {
         'scripts/notice',
         '--validate'
       ]
-    }
+    },
+
+    apiIntegrationTests: {
+      cmd: process.execPath,
+      args: [
+        'scripts/functional_tests',
+        '--config', 'test/api_integration/config.js',
+        '--es-from', 'source',
+        '--bail',
+      ],
+    },
+
+    functionalTests: {
+      cmd: process.execPath,
+      args: [
+        'scripts/functional_tests',
+        '--config', 'test/functional/config.js',
+        '--es-from', 'source',
+        '--bail',
+        '--',
+        '--server.maxPayloadBytes=1648576',
```

Comment:
Yup, this is a confusing one, THere's one test in management that uses the server.maxPayloadBytes setting. It's happening here: https://github.com/elastic/kibana/pull/18967/files#diff-4d343b812281fca44b2afeab2d3f9e76L70. This test will fail locally as well, unless you start Kibana with that setting. (added by rashmi)

https://github.com/elastic/kibana/blob/master/tasks/config/run.js#L153 is where dev_mode.enabled=false is being set. it's needed for the markdown vis test to pass. (been talking to tim roes about this)

https://github.com/elastic/kibana/pull/18967/files#diff-4d343b812281fca44b2afeab2d3f9e76L44 is where we do env.name instead of just env. I looked at where env is being used currently and it looks like we still support env=development, but prefer env.name=development.